Documents/Automation/EncryptRootVolume/Documents/npark-encryptrootvolume.json

```diff
@@ -187,10 +206,10 @@
       "inputs": {
         "DocumentName": "AWS-DetachEBSVolume",
         "RuntimeParameters": {
-          "VolumeId": "{{describeInstanceRootVolume.RootDeviceVolumeId}}"
+          "VolumeId": "{{describeInstanceRootVolume.rootDeviceVolumeId}}"
         }
       },
-      "name": "DetachEBSVolume",
+      "name": "detachEBSVolume",
```

Comment:
I usually prefer the "name" property at the top. You are using a mix of positions for this property, pick one for consistency and readability.

Reply:
corrected in the latest commit.